extensions/wikidata/module/langs/translation-en.json

```diff
@@ -148,6 +148,8 @@
     "warnings-messages/multi-valued-property-is-required-for-new-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}.",
     "warnings-messages/multi-valued-property-is-required-for-existing-item/title": "{property_entity} should have more than one statement on existing items.",
     "warnings-messages/multi-valued-property-is-required-for-existing-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}. If the item already has statements with this property in Wikidata, then this warning can be ignored.",
+    "warnings-messages/values-should-not-be-used-as-qualifier/title": "{property_entity} have qualifiers with inappropriate values.",
+    "warnings-messages/values-should-not-be-used-as-qualifier/body": "{property_entity} have qualifier with {added_property_entity} have in appropriate values. for instance on {example_entity}.",
```

Comment:
The constraint body does not look grammatical to me. The variable names are not ideal either, I would replace `added_property_entity` by `qualifier_entity` and `property_entity` by `statement_entity` to emphasize their roles. How about this for the title and body:
`Invalid value for qualifier {qualifier_entity} on statement {statement_entity}`
`On items such as {example_entity}, an invalid value for qualifier {qualifier_entity} was supplied on statement {statement_entity}.`